package.json

```diff
@@ -91,6 +91,7 @@
     "blockstore-datastore-adapter": "^4.0.0",
     "datastore-core": "^8.0.2",
     "datastore-level": "^9.0.4",
+    "emittery": "^1.0.1",
```

Comment:
Would it be possible to use our event emitter (https://github.com/fission-codes/webnative/blob/main/src/common/event-emitter.ts) instead of using emittery?

Reply:
How did I miss that? 😅 Yeah sure thing! I'll rename `removeListener` to `off` though so that it matches `on`.

Reply:
The name `removeListener` is a better than `off` in my opinion. I've always read the `on` as "on this event, do this thing". The `off` doesn't really make sense in this context, but `removeListener` is explicit about what you are doing.

That said, I'd be alright with either one. We modeled this off of the node event emitter, and they define [emitter.off](https://nodejs.org/api/events.html#emitteroffeventname-listener) as an alias for `removeListener`.

Reply:
I’ll define the ‘on’ and ‘off’ aliases for ‘addListener’ and ‘removeListener’ respectively.